www/guides/create_account.md

```diff
@@ -0,0 +1,186 @@
+---
+sidebar_position: 8
+---
+# Create account
+Since there are no Externally Owned Accounts (EOA) in StarkNet, all Accounts in StarkNet are contracts.
+
+Unlike in Ethereum where a wallet is created with a public and private key pair, StarkNet Accounts are the only way to sign transactions and messages, and verify signatures. Therefore a Account - Contract interface is needed.
+
+Account contracts on StarkNet cannot be deployed without paying a fee.
+Create an account is a bit tricky ; you have several steps :
+1. Decide on your account type (OpenZeppelin, Argent, ...).
+2. Compute the address of your future account.
+3. Send funds to this pre-computed address. The funds will be used to pay for the account contract deployment, and remains will fund the new account.
+4. Actual deployment of the Account
+
+## Create OZ (Open Zeppelin) account :
+Here, we will create a wallet with the Open Zeppelin smartcontract v0.5.1. The contract class is already implemented in both Testnet 1 & 2.  
+```typescript
+import { Account, ec, json, stark, Provider, hash } from "starknet";
+```
+### compute address :
+```typescript
+// connect provider
+const provider = new Provider({ sequencer: { network: "goerli-alpha" } });
+
+// new Open Zeppelin account v0.5.1 :
+    // Generate public and private key pair.
+const privateKey = stark.randomAddress();
+console.log('New OZ account :\nprivateKey=', privateKey);
+const starkKeyPair = ec.getKeyPair(privateKey);
+const starkKeyPub = ec.getStarkKey(starkKeyPair);
+console.log('publicKey=', starkKeyPub);
+
+const OZaccountClashHass = "0x2794ce20e5f2ff0d40e632cb53845b9f4e526ebd8471983f7dbd355b721d5a";
+// Calculate future address of the account
+const OZaccountConstructorCallData = stark.compileCalldata({ publicKey: starkKeyPub });
+const OZcontractAddress = hash.calculateContractAddressFromHash(
+    starkKeyPub, 
+    OZaccountClashHass, 
+    OZaccountConstructorCallData, 
+    0
+);
+console.log('Precalculated account address=', OZcontractAddress);
+
+```
+If you want a specific private key, replace `stark.randomAddress()` by your choice.
+
+Then you have to fund this address.  
+How to proceed is out of the scope of this guide, by you can for example :
+- transfert ETH from an other wallet.
+- Bridge ETH to this Starknet address.
+- Use a faucet.
+- Mint ETH on starknet-devnet.
```

Comment:
I would maybe expand this part a bit, like from:
https://github.com/0xs34n/starknet.js/pull/480/files#diff-c469c62c8cbadad5458c58336bc2394c2c7e8e25bcf1fea0c652bc2deccdb323L83

Reply:
OK

Reply:
Oh, this link I made doesn't make sense anymore... suggestion what I meant:
```suggestion
- transfert ETH from an other wallet.
- Bridge ETH to this Starknet address.
- Use a faucet. (https://faucet.goerli.starknet.io/)
- Mint ETH on starknet-devnet, like so:
```bash
curl -X POST http://127.0.0.1:5050/mint -d '{"address":"0x04a093c37ab61065d001550089b1089922212c60b34e662bb14f2f91faee2979","amount":50000000000000000000,"lite":true}' -H "Content-Type:application/json"
// {"new_balance":50000000000000000000,"tx_hash":null,"unit":"wei"}
```
```